doc/source/Resources/Contributing.rst

```diff
@@ -135,18 +135,24 @@ Documentation
 Documentation for PyAEDT is generated from three sources:
 
 - Docstrings from the classes, functions, and modules of PyAEDT using `sphinx.ext.autodoc`.
-- reStructuredText (RST) files from `docs/`
+- reStructuredText (RST) files from `doc/`
```

Comment:
Question: What is the difference between using `docs/` and ``docs/``? I've always used two `` marks. I thought one meant 
 it was a variable.

Reply:
I made a mistake.  Restructured needs two "`", markdown can get away with one.  Reason for this is the backquotes are used in other directives or links.